capnpc-go/capnpc-go.go

```diff
@@ -615,6 +605,20 @@ func (g *generator) defineField(n *node, f field) (err error) {
 				return err
 			}
 		}
+
+		switch t.AnyPointer().Which() {
+		// Unconstrained?
```

Comment:
Since there's only one `case` here, this should probably just be an if statement. Which I think would also obviate this comment, which feels excessive to me. Same for the nested switch, and elsewhere where this pattern gets used.

Reply:
My intent was to progressively add the other cases over the next few weeks.  I suppose I can always revert to a `switch` statement at that time, though.